Documentation/botr/readytorun-format.md

```diff
@@ -425,6 +494,27 @@ The module override index translation algorithm is as follows (**ILAR** = *the n
 
 **TODO**: document attribute presence encoding
 
+## READYTORUN_SECTION_ASSEMBLIES (v3.2+)
+
+This section is only present in composite R2R files. It is a straight binary array of the entries `READYTORUN_SECTION_ASSEMBLIES_ENTRY` parallel to the indices in the manifest metadata AssemblyRef table in the sense that it's a linear table where the row indices correspond to the equivalent AssemblyRef indices. Just like in the AssemblyRef ECMA 335 table, the indexing is 1-based (the first entry in the table corresponds to index 1).
+
+```C++
+struct READYTORUN_SECTION_ASSEMBLIES_ENTRY
+{
+    DWORD  CorHeaderRva;          // RVA of the input MSIL metadata COR header
+    DWORD  AvailableTypesRva;     // RVA of the available types table
```

Comment:
The type used for RVA + SIZE is `IMAGE_DATA_DIRECTORY`

It is likely that the number of per-assembly entries will grow. It may be nice to use the same ID+ `IMAGE_DATA_DIRECTORY` like what is used in the regular R2R header today.

Reply:
Good point, updated in 7th commit.